Lib/test/test_memoryview.py

```diff
@@ -394,6 +394,20 @@ def test_issue22668(self):
         self.assertEqual(c.format, "H")
         self.assertEqual(d.format, "H")
 
+    def test_minimal_buffer(self):
```

Comment:
This test is repeated in 6 test classes:
```
test_minimal_buffer (test.test_memoryview.ArrayMemorySliceSliceTest) ... ok
test_minimal_buffer (test.test_memoryview.ArrayMemorySliceTest) ... ok
test_minimal_buffer (test.test_memoryview.ArrayMemoryviewTest) ... ok
test_minimal_buffer (test.test_memoryview.BytesMemorySliceSliceTest) ... ok
test_minimal_buffer (test.test_memoryview.BytesMemorySliceTest) ... ok
test_minimal_buffer (test.test_memoryview.BytesMemoryviewTest) ... ok
```
Move it to more appropriate place, so that it will not be repeated. For example in OtherTest.

Also, this test is not crashed on Python 3.

Add a reference to the issue.